OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -93,6 +93,30 @@ public override Activity Tick(Actor self)
 			var targetAltitude = aircraft.CenterPosition.Z + aircraft.Info.CruiseAltitude.Length - self.World.Map.DistanceAboveTerrain(aircraft.CenterPosition).Length;
 			if (aircraft.CenterPosition.Z < targetAltitude)
 				desiredFacing = aircraft.Facing;
+			else
+			{
+				// Using the turn rate, compute a hypothetical circle traced by a continuous turn.
+				// If it contains the destination point, it's unreachable without more complex manuvering.
+
+				// TurnSpeed -> divide into 256 to get the number of ticks per rotation
+				// MovementSpeed -> multiply to get distance travelled per rotation (circumference)
+				// Multiply by magic 2*pi integer ratio to get the turn radius (1024/41728==2*pi*256)
+				var turnRadius = aircraft.MovementSpeed * 41728 / (aircraft.TurnSpeed * 1024);
```

Comment:
We approximate this as `45 * speed / aircraft.Info.TurnSpeed;` in the `ReturnToBase`, including extra leeway.  This would be a good idea here too because the target might be moving, and it would be good to keep the code for evaluating these turns consistent if not actually shared.

Reply:
Okay, I moved ReturnToBase.CalculateTurnRadius to a public static on Fly, where it is shared.